stage1/rootfs/usr_from_host/Makefile

```diff
@@ -1,13 +1,19 @@
 # do not include systemd in stage1, let's find it at run-time
 
-usr.done install: Makefile
+usr.done: Makefile install
+	cp -f install ../aggregate/install.d/00usr
+	touch usr.done
```

Comment:
Ok, I am removing the `touch usr.done`.

And since the file `usr.done` is not created anymore, I added `usr.done` in the phony list so that it is explicit instead of implicit.